slang-gfx.h

```diff
@@ -2266,6 +2266,8 @@ class IDevice : public ISlangUnknown
         ShaderCacheDesc shaderCache = {};
         // Configurations for Slang compiler.
         SlangDesc slang = {};
+        /// Whether to enable ray tracing validation (currently only Vulkan - D3D requires app layer to use NVAPI)
```

Comment:
For binary compatibility, all new fields must be added as an extendedDesc instead of directly inside the struct.

Reply:
Hi Yong - thanks - didn't see that mechanism for descriptors. Latest commit matches the pattern used for the D3D 12 device extended descriptor.